pkg/statsd/handler_http_forwarder_v2.go

```diff
@@ -368,18 +388,6 @@ func (hfh *HttpForwarderHandlerV2) serialize(message proto.Message) ([]byte, err
 	return buf, nil
 }
 
-// debug rendering
```

Comment:
Nixed this as it wasn't be used and should have been cleaned up.

Reply:
Please restore this - it's deliberately here for debug purposes.

Reply:
It's shift in its spot but it is back for when you need it.

Reply:
I mean... I wanted it back so it doesn't conflict with my branch.

Now I have two conflicts :|

Reply:
Ah, my bad.